### ISSUE TITLE
Binary Score

### ISSUE THREAD

**Issue by albths** [keisen/tf-keras-vis#78]
Hi,

First of all, thanks a lot for this great visualization toolbox. I am using Efficientnet to categorize images into a binary score (i. e. only one output neuron). Can you please provide an end-to-end example, how to use the class BinaryScore in this context? SmoothGrad works fine in my case, but somehow GradCam and ScoreCam  only yield broken results, even though I copied your example code. Thanks a lot in advance!

**Comment by albths** [comment]
This is exactly, what I was looking for. Thanks a lot. Somehow didn't see these instructions.

However, it seems that using BinaryScore for negative class (i. e. score = Binary(0.0)) produces very strange images with a grid-like (non-informative) feature representation. Would be grateful for any help on that.

**Comment by aguirrejuan** [comment]
Hi, first of all, thanks for the great job on this repo. 

I am presenting a similar issue with just one output (sigmoid). The issue seems to be happening for the `K.softmax`, this is because when one value (which is the case) is given, the returned value is always one. 

https://github.com/keisen/tf-keras-vis/blob/0bc00a80c04a66669df8dfd8d1137cdca9b86610/tf_keras_vis/scorecam.py#L165-L166

Also, I think the operation of softmax must be over channels instead of logits, the paper says  

![image](https://user-images.githubusercontent.com/50622777/164246207-e3f57c09-8869-4bac-8960-899c7d302eeb.png)

Where softmax is applied over the results for a class in specific.
